### ISSUE TITLE
No more additional components?

### ISSUE THREAD

**Issue by murugappanrm** [quasarcast/takeoff#1]
Hi Luke,

Looks like this project has been abandoned mid-way. I have been trying to add more myself but failed to add. Perhaps, if you do not have the time, please show us how to add the videos and the playground items. We are newbies. We thought this will be helpful but unfortunately this too remains incomplete.

Please advise.

**Comment by ldiebold** [comment]
New videos have been added.

I apologise for the delay, and will get better at this. Sometimes life deals punches and things get shaky for a while :sweat_smile:

**Comment by murugappanrm** [comment]
Hi Luke,
Thank you once again for a great job well done. I am learning more about Quasar than the documentation provides. Just an aside note, there are a number of other videos that that we, newbies, find it hard to follow and as a result unable to contribute or help with your views-count.  One such is the "Goal Setting App". We assumed it was a full tutorial of using Quasar to develop an app. But when we started the first video we found that it just started suddenly with an installed Quasar framework and we got lost. So sorry, that is newbies like us.

Just a bit background about "us". We are are large group of developers in the ASEAN (Association of South East Asian Nations - Thailand, Indonesia, Malaysia, Vietnam, Philippines, Singapore and Myanmar). We sit a virtual class room and go through all these courses and learn. We exchange ideas, views, feedback and also produce feedback on what we should use and not use. I hope i have manage make where we newbies come from. Our goal is to build a huge group technology developers who will contribute to IT within the region. Hopefully one day, we will be able to create and distribute really free and open-source apps for the medical, engineering and business community. In short, we are ASEAN group of business solution developers. We just developed a complete medical claims auditing and billing system which is a back-end app to constrain the scrupulous claims made by the medical profession.

**Comment by ldiebold** [comment]
Okay, will try to keep newbies in mind and direct them to beginner material in the future!

It's also important to note that those using Quasar should first be familiar with both Vue3 and Javascript. Vue3 is built on Javascript, and Javascript is built on Vue3. Since there are many many tutorials out there targeting Javascript/Vue, I intentionally only teach Quasar.

Perhaps we need a video on the home page of take off describing this, and directing people to those resources?

**Comment by murugappanrm** [comment]
@ldiebold ,

Hi Luke, 

I know this thread may not be the right one but i am posting here as the issue is the same. I have recently been going over all the great courses you have created and i found one list seems to be hanging. The course is titled "Quasar Vue's Qtable". This one is supposed to have 6 videos but at this date the last one seems to be 4/6. What happen to 5 and 6/6??  I am afraid i may start the course and then at end of 4/6 i would be left hanging! 

Please advise.
